server/game/cards/characters/04/jaqenhghar.js

```diff
@@ -37,7 +37,7 @@ class JaqenHGhar extends DrawCard {
         });
         this.forcedReaction({
             when: {
-                onCardLeftPlay: (event, player, card) => card === this
+                onCardLeftPlay: event => event.card === this
```

Comment:
FYI, Jaqen is broken and will still be broken after this PR. The forced reaction will not (and should not) fire because it would be a reaction to itself leaving play - you can only interrupt self-leaving play. It also should not be implemented as an interrupt / reaction at all because it isn't something that can be cancelled (e.g. you can't Treachery getting rid of the tokens). Probably should open up a separate issue about this cause I keep forgetting about it.